rest_framework/compat.py

```diff
@@ -33,6 +33,14 @@ def unicode_to_repr(value):
     return value
 
 
+def total_seconds(timedelta):
+    # TimeDelta.total_seconds() is only available in Python 2.7
+    if hasattr(timedelta, 'total_seconds'):
+        return timedelta.total_seconds()
+    else:
+        return (timedelta.days * 86400) + timedelta.seconds + (timedelta.microseconds / 1000000.0)
```

Comment:
Further thoughts - let's make it more explicit that we're returning a float value...
 
   (timedelta.days * 86400.0) + float(timedelta.seconds) + (timedelta.microseconds / 1000000.0)